lib/junit.cf.mustache

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Yeah thats what I was thinking. Thoughts?

I have done similar in cfe_internal/enterprise/templates